datarobot_batch_scoring/network.py

```diff
@@ -169,8 +169,8 @@ def _response_callback(self, r, batch=None, *args, **kw):
                 self.ui.error(msg)
                 self.send_error_to_ctx(batch, msg)
             else:
-                self.ui.warning('respooling batch {}'
-                                .format(batch.id))
+                self.ui.warning('retrying failed batch {}, attempts left: {}'
```

Comment:
It's more recommended way to use logging formatting style, i.e:

```
self.ui.warning('retrying failed batch %s, attempts left: %d', batch.id, batch.rty_cnt - 1)
```

The idea is to do not initiate string formatting if logging is configured to print only certain log levels. Though, in this case it won't be a performance bottleneck and I'm ok with it. Just FYI. :)